float64[] rmat
float64[] tvec
uint16 id
uint16 num_aruco_ids_detect